src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/config/RedisStreamEventControllerConfigs.kt

```diff
@@ -6,13 +6,9 @@ import org.springframework.context.annotation.Configuration
 
 @Configuration
 class RedisStreamEventControllerConfigs(
-  @Value("\${redisStream.eventController.streamKey}") val streamKey: String,
-  @Value("\${redisStream.eventController.consumerGroupPrefix}") consumerGroupPrefix: String,
-  @Value("\${redisStream.eventController.consumerNamePrefix}") consumerNamePrefix: String,
-  @Value("\${redisStream.eventController.failOnErrorCreatingConsumerGroup}")
-  val faiOnErrorCreatingConsumerGroup: Boolean
+  @Value("\${redisStream.eventController.streamKey}") val streamKey: String
 ) {
-  private val uniqueConsumerId = UUID.randomUUID().toString()
-  val consumerGroup = "$consumerGroupPrefix-$uniqueConsumerId"
-  val consumerName = "$consumerNamePrefix-$uniqueConsumerId"
+  private val uniqueInstanceId = UUID.randomUUID().toString()
+
+  val instanceId: String = "instance-$uniqueInstanceId"
```

Comment:
```suggestion
  val instanceId: String = "event-dispatche-$uniqueInstanceId"
```

this instance id field replaces the consumer name value, still used to save entities in Redis.
In those cases is better to have the same output value, just to be sure to not create problems in others systems that might read those values (f.e. deployment pipeline)

Reply:
Good point, addressed with [0ef3a86](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/pull/370/commits/0ef3a86d776e0867efd63cceb6f8cdd7582cb671)